clean.cmd

```diff
@@ -8,12 +8,13 @@ for /f "tokens=2 delims=," %%F in ('tasklist /nh /fi "imagename eq VBCSCompiler.
 :: compare to match as many versions of "all" that we can
 :: All other argument validation happens inside Run.exe
 set NO_DASHES_ARG=%1
+if not defined NO_DASHES_ARG goto no_args
 if /I [%NO_DASHES_ARG:-=%] == [all] (
   echo Cleaning entire working directory ...
   call git clean -xdf
   exit /b !ERRORLEVEL!
 )
-
+:no_args
 if [%1]==[] set __args=-b
```

Comment:
You could just move this up above and use __args instead of introducing the goto.